pom.xml

```diff
@@ -154,6 +154,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.infradna.tool</groupId>
+      <artifactId>bridge-method-annotation</artifactId>
+      <version>1.29</version>
+    </dependency>
```

Comment:
This is delivered by Jenkins core, so this dependency isn't needed. While in general it is a best practice to declare direct dependencies rather than consuming a transitive dependency without declaring it, with Jenkins plugins we usually recommend the opposite (not declaring a dependency and consuming it transitively via core) in order to avoid bloating JPIs with unused JARs in `WEB-INF/lib` and dealing with Enforcer issues.

Reply:
It wasn't in my classpath when I did the implementation.